PocketBeagle/Grove/Lesson-1.Control-the-Light/README.md

```diff
@@ -0,0 +1,29 @@
+# Description:
+
+In this lesson, students will light up the RGB LED, and learn how to use Slide Potentiometer and Rotary Angle Sensor to change the light of RGB LED.
+
+## Hardware Requirement:
```

Comment:
Do we need a separate requirement list given that the setup is simply connecting the required devices? Do we need instructions on printing out the tutorial at the very beginning and encouraging users to shut the power down before making connections?